### ISSUE TITLE
Unexpected stats for certain SSPXR modules (1.5.1)

### ISSUE THREAD

**Issue by LouisB3** [GarwelGarwel/KerbalHealth#138]
New issue as I don't think I can reopen #114. Upon review of the stock and SSPXR values for KH 1.5.1, here is everything unexpected that I noticed. 

- Some parts have their lounge crew capacity explicitly defined, while others do not (I think this means they apply to everyone without limit). The parts without a defined lounge capacity are:
    - Stock:
        - Hitchhiker
        - Mk1 Cabin
        - Mk2 Cabin
        - Mk3 Cabin
    - SSPXR:
        - 1.25m habitat
        - 1.25m centrifuges (x2)
- The 3.75m radiation shelter, compared to the short 3.75m habitat, offers more space or a better confinement multiplier, but not better radiation shielding. It should be the reverse as it sacrifices habitable space for its resource tanks, which provide shielding according to the part description.
- Centrifuges have some unexpected or inconsistent features:
    - Unusually good confinement multipliers when used as a lounge (.16-.18, compared to .30 to .33 typically.)
    - No differentiation between sizes for the microgravity multiplier; previously, large rings provided better multipliers to correspond with their greater artificial gravity. (Crew capacity is a weaker differentiator as there is infrequently a reason to send more than a couple of kerbals anywhere.)
    - Microgravity crew capacity is specified for the 2.5m and 3.75m rings but not the 1.25m rings.
    - The 1.25m and 2.5m inflatable rings can hold radiation shielding, unlike other inflatables and the 3.75m centrifuges.
- Cupolas have lost their recuperation! This includes the 1.25m cupola, 2.5m observation module, and 3.75m cupola. (The 2.5m observation module also is still a lounge.)
- Medical bays have some inconsistency in their crew capacity. Per the patch notes, they should now provide vessel-wide benefits; however:
    - The stock MPL is still marked "partCrewOnly = true"
    - The 3.75m lab is marked "crewCap = 3"

**Comment by yalov** [comment]
I already have mentioned some time ago about utilizing greenhouses in the case without any LifeSupport mod, just putting my patches there

```
@PART[sspx-greenhouse-25-1]:NEEDS[StationPartsExpansionRedux&!Snacks&!USILifeSupport&!TacLifeSupport]:AFTER[StationPartsExpansionRedux]
{
	%RESOURCE[RadiationShielding]
	{
		%name = RadiationShielding
		%maxAmount = 10
	}

	MODULE
	{
		name = ModuleKerbalHealth
		multiplyFactor = Loneliness
		multiplier = 0.5
	}

	MODULE
	{
		name = ModuleKerbalHealth
		recuperation = 0.5
		crewCap = 18
		resourceConsumptionPerKerbal = 1
	}
}


@PART[sspx-greenhouse-375-1]:NEEDS[StationPartsExpansionRedux&!Snacks&!USILifeSupport&!TacLifeSupport]:AFTER[StationPartsExpansionRedux]
{
	%RESOURCE[RadiationShielding]
	{
		%name = RadiationShielding
		%maxAmount = 15
	}

	MODULE
	{
		name = ModuleKerbalHealth
		multiplyFactor = Loneliness
		multiplier = 0.5
	}

	MODULE
	{
		name = ModuleKerbalHealth
		recuperation = 0.75
		crewCap = 12
		resourceConsumptionPerKerbal = 1
	}
}


@PART[sspx-aquaculture-375-1]:NEEDS[StationPartsExpansionRedux&!Snacks&!USILifeSupport&!TacLifeSupport]:AFTER[StationPartsExpansionRedux]
{
	// already there
	//MODULE
	//{
	//	name = ModuleKerbalHealth
	//	shielding = 3
	//}

	%RESOURCE[RadiationShielding]
	{
		%name = RadiationShielding
		%maxAmount = 15
	}

	MODULE
	{
		name = ModuleKerbalHealth
		multiplyFactor = Loneliness
		multiplier = 0.5
	}
}
```

**Comment by GarwelGarwel** [comment]
Fixed in v1.5.2.
